features/sample_manifest/sample_manifest.feature

```diff
@@ -388,7 +388,7 @@ Feature: Sample manifest
     Examples:
       | initial                           | update                            | state     | message                                                           |
       | sample_manifest_a_accessions.csv  | sample_manifest_no_accessions.csv | Completed | Sample manifest successfully uploaded                             |
-      | sample_manifest_a_accessions.csv  | sample_manifest_b_accessions.csv  | Completed | The accession number does not match the existing accession number |
+      | sample_manifest_a_accessions.csv  | sample_manifest_b_accessions.csv  | Failed    | The accession number does not match the existing accession number |
```

Comment:
Wuh? Well that was almost certainly my fault. As I both wrote the original code, and ported that feature across.